pyaedt/edb.py

```diff
@@ -1356,36 +1356,122 @@ def solve_siwave(self):
         return True
 
     @pyaedt_function_handler()
-    def add_design_variable(self, variable_name, variable_value):
+    def add_design_variable(self, variable_name, variable_value, is_parameter=False):
         """Add a design variable.
 
         Parameters
         ----------
         variable_name : str
-            Name of the variable.
+            Name of the variable, if it begins by a "$", it will be a "Project Variable".
         variable_value : str, float
             Value of the variable with units.
+        is_parameter : bool, optional
+            True to add a "Parameter Default", False for a "Local Variable"
```

Comment:
```suggestion
            Whether the variable is a parameter. The default is ``True``. If ``False``, the parameter is
            added as a local variable.
```

Reply:
I'm not sure if the edit I made above is technically accurate, but at least it does show the format that we want to use.

Reply:
I change it to : 
-The default is ``True`` to add the variable as a parameter default. If ``False``, the variable is added as a local variable.
It is more technically accurate.

Reply:
@jeremich41 I edited once again based on your reply. Thanks!